.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    "es2015-native-modules"
```

Comment:
I'm curious about this. Which should I recommend that people use? The `-webpack` or `-native-modules` preset?